sample-workflows/transifex-pull.yml

```
@@ -0,0 +1,61 @@
+name: Pull Translations from Transifex
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  push:
+    branches:
+      - '*'
+  workflow_dispatch:
+
+jobs:
+  update-translation:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        version: [ 3.13 ]
```

python-newest now reflects the 3.14 release.


```suggestion
        version: [ '3.14' ]
```